test/hotspot/gtest/opto/test_regmask.cpp

```diff
@@ -83,11 +83,11 @@ TEST_VM(RegMask, Set_ALL) {
   // Check that Set_All doesn't add bits outside of CHUNK_SIZE
```

Comment:
The comment refers to CHUNK_SIZE, which no longer exists.

Reply:
Thanks, I'll fix it (and grep for other occurrences)